_docs/concepts/security/authn-policy.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Istio Authentication Policy
+overview: Describes Istio authentication policy
+
+order: 10
+
+layout: docs
+type: markdown
+---
+{% include home.html %}
+
+## Overview
```

Comment:
You can remove the Overview header, there's already a title overlayed above by the web site.

Reply:
Done